tests/test_multipart_encoder.py

```diff
@@ -81,6 +89,44 @@ def test_accepts_encoded_strings_with_unicode(self):
         assert self.instance.read() == s
 
 
+class TestFileFromURLWrapper(unittest.TestCase):
+    def setUp(self):
+        s = requests.Session()
+        self.recorder = get_betamax(s)
+
+    def test_read_file(self):
+        url = ('https://stxnext.com/static/img/logo.830ebe551641.svg')
+        with self.recorder.use_cassette(
+                'file_for_download', **preserve_bytes):
+            self.instance = FileFromURLWrapper(url)
+            assert self.instance.len == 5177
+            chunk = self.instance.read(20)
+            assert chunk == b'<svg xmlns="http://w'
+            assert self.instance.len == 5157
+            chunk = self.instance.read(0)
+            assert chunk == b''
+            assert self.instance.len == 5157
+            chunk = self.instance.read(10)
+            assert chunk == b'ww.w3.org/'
+            assert self.instance.len == 5147
+
+    def test_no_content_length_header(self):
+        url = (
+            'https://api.github.com/repos/sigmavirus24/github3.py/releases/'
+            'assets/37944'
+        )
+        with self.recorder.use_cassette(
+                'stream_response_to_file', **preserve_bytes):
```

Comment:
Can you add `tests/cassettes/stream_response_to_file.json`? I'm not seeing that file in this pull request.

Reply:
it's already there [link](https://github.com/requests/toolbelt/blob/master/tests/cassettes/stream_response_to_file.json) :)

Reply:
Hah! Well done, reusing that cassette :)